circle.yml

```diff
@@ -4,6 +4,18 @@ machine:
   java:
     version: oraclejdk8
 
+dependencies:
```

Comment:
Can you put a note here linking to See: https://circleci.com/docs/1.0/language-scala/ to remove this block when CircleCI supports newer SBT versions?

Reply:
Sure. :)